new_arr.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'tax_calculate'
+module New
+  obj = Tax.new
+  Taxes = obj.ret_tax
+  New_arr = []
+  @@j = 0
+  def disp(arr)
+    for i in (0...arr.length)
+      y = ""
+      x = arr[i].split(/ /)
+      for j in (0...x.length-2)
+        y = y + x[j] + " "
+      end
```

Comment:
Use meaning-full variable, classes, modules, method names. Please fix this in the whole code.